### PR TITLE
fix: Align module name in go.mod

### PR DESCRIPTION
**Pull request by tchap**
Hmm, forgot to update this. Go modules 🤦

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/casbin/zap-logger
+module github.com/casbin/zap-logger/2
```
Comment (hsluoyz) [inline go.mod:1]: 2 or v2?

Comment (tchap) [reply go.mod]: Amended.

**Comment by tchap** [comment]
I am going off now, so in case you merge, don't forget to tag `v2.0.1`. Thank you for the fast response 😉 🙇

**Comment by hsluoyz** [comment]
Re-tagged v2.0.0 to this PR: https://github.com/casbin/zap-logger/releases/tag/v2.0.0

**Pull request merged**